client/src/app/pages/applications/applications-table-assessment/applications-table-assessment.tsx

```diff
@@ -566,160 +491,316 @@ export const ApplicationsTable: React.FC = () => {
+                title={t("composed.noDataStateTitle", {
+                  what: t("terms.applications").toLowerCase(),
+                })}
+                description={t("composed.noDataStateBody", {
+                  what: t("terms.application").toLowerCase(),
+                })}
+              />
+            }
+            numRenderedColumns={numRenderedColumns}
+          >
+            {currentPageItems?.map((application, rowIndex) => {
+              return (
+                <Tbody
+                  style={{ cursor: "pointer" }}
+                  key={application.name}
+                  onClick={(event) => {
+                    handlePropagatedRowClick(event, () => {
+                      if (activeAppInDetailDrawer === application) {
+                        closeDetailDrawer();
+                      } else {
+                        openDetailDrawer(application);
                       }
+                    });
+                  }}
+                >
+                  <Tr>
+                    <TableRowContentWithControls
+                      {...tableControls}
+                      item={application}
+                      rowIndex={rowIndex}
                     >
-                      {t("actions.review")}
-                    </Button>
-                  </ToolbarItem>
-                </RBAC>
-                {dropdownItems.length ? (
-                  <ToolbarItem>
-                    <KebabDropdown
-                      dropdownItems={dropdownItems}
-                    ></KebabDropdown>
-                  </ToolbarItem>
-                ) : (
-                  <></>
-                )}
-              </ToolbarGroup>
-            </>
-          }
-          noDataState={
-            <NoDataEmptyState
-              // t('terms.applications')
-              title={t("composed.noDataStateTitle", {
-                what: t("terms.applications").toLowerCase(),
-              })}
-              // t('terms.application')
-              description={t("composed.noDataStateBody", {
-                what: t("terms.application").toLowerCase(),
-              })}
-            />
-          }
+                      <Td
+                        width={20}
+                        {...getTdProps({ columnKey: "name" })}
+                        modifier="truncate"
+                      >
+                        {application.name}
+                      </Td>
+                      <Td
+                        width={25}
+                        {...getTdProps({ columnKey: "description" })}
+                        modifier="truncate"
+                      >
+                        {application.description}
+                      </Td>
+                      <Td
+                        width={10}
+                        modifier="truncate"
+                        {...getTdProps({ columnKey: "businessService" })}
+                      >
+                        <TableText wrapModifier="truncate">
```

Comment:
I don't think you need the `TableText` here if you're using the `Td modifier="truncate"`. I think `TableText` is only necessary when you need its other props as well and the built-in modifier on the `Td` is enough if it works. (at least that's what I can glean from the ["Controlling text" example](https://www.patternfly.org/components/table#controlling-text) which doesn't use TableText and the ["Modifiers with table text" example ](https://www.patternfly.org/components/table#modifiers-with-table-text)which does... I'm not sure exactly)